ui/pages/onboarding-flow/secure-your-wallet/secure-your-wallet.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Box from '../../../components/ui/box';
+import Button from '../../../components/ui/button';
+import Typography from '../../../components/ui/typography';
+import {
+  TEXT_ALIGN,
+  TYPOGRAPHY,
+  JUSTIFY_CONTENT,
+  FONT_WEIGHT,
+} from '../../../helpers/constants/design-system';
+import ProgressBar from '../../../components/app/step-progress-bar';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { ONBOARDING_REVIEW_SRP_ROUTE } from '../../../helpers/constants/routes';
+import SkipSRPBackup from './skip-srp-backup-popover';
+
+export default function SecureYourWallet() {
+  const history = useHistory();
+  const t = useI18nContext();
+  const [showSkipSRPBackupPopover, setShowSkipSRPBackupPopover] = useState(
+    false,
+  );
+
+  const handleClickRecommended = () => {
+    history.push(ONBOARDING_REVIEW_SRP_ROUTE);
+  };
+
+  const handleClickNotRecommended = () => {
+    setShowSkipSRPBackupPopover(true);
+  };
+
+  const subtitles = {
+    en: 'English',
+    es: 'Spanish',
+    hi: 'Hindi',
+    id: 'Indonesian',
+    ja: 'Japanese',
+    ko: 'Korean',
+    pt: 'Portuguese',
+    ru: 'Russian',
+    tl: 'Tagalog',
+    vi: 'Vietnamese',
+  };
+
+  return (
+    <div className="secure-your-wallet">
+      {showSkipSRPBackupPopover && (
+        <SkipSRPBackup handleClose={() => setShowSkipSRPBackupPopover(false)} />
+      )}
+      <ProgressBar stage="SEED_PHRASE_VIDEO" />
+      <Box
+        justifyContent={JUSTIFY_CONTENT.CENTER}
+        textAlign={TEXT_ALIGN.CENTER}
+        marginBottom={4}
+      >
+        <Typography variant={TYPOGRAPHY.H2} fontWeight={FONT_WEIGHT.BOLD}>
+          {t('seedPhraseIntroTitle')}
+        </Typography>
+      </Box>
+      <Box
+        justifyContent={JUSTIFY_CONTENT.CENTER}
+        textAlign={TEXT_ALIGN.CENTER}
+        marginBottom={6}
+      >
+        <Typography
+          variant={TYPOGRAPHY.H4}
+          className="secure-your-wallet__details"
+        >
+          {t('seedPhraseIntroTitleCopy')}
+        </Typography>
+      </Box>
+      <Box>
+        <video controls style={{ borderRadius: '10px' }}>
+          <source
+            type="video/webm"
+            src="./images/videos/recovery-onboarding/video.webm"
+          />
+          {Object.keys(subtitles).map((key) => {
+            return (
+              <track
+                default={Boolean(key === 'en')}
```

Comment:
Ideally this would default to whatever the current locale is. We don't want to show subtitles the user doesn't understand.

Maybe we could use `en` as a fallback, if the current locale doesn't have subtitles? :thinking: I'm not sure whether showing English subtitles or showing none at all is better in that case.

Reply:
@rachelcope Should we show English subtitles as a default if we don't have a subtitle track for the locale of the user?